reference/variables.md

```diff
@@ -202,6 +203,30 @@ As HTTP request headers are defined to be case-insensitive, their names are lowe
 $request/headers/user-agent
 ```
 
+If a client URL path is matched by a [wildcard path](/reference/OpenAPI/differences.md#wildcard-paths), `$request/endpoint` is the path part preceding the part matched by `/**`. Otherwise, `$request/endpoint` is the same as `$request/path`.
+
+```yaml
+…
+basePath: /api
+…
+paths:
+  /**:
+    …
+  /foo/**:
+    …
+  /foo/qux:
+    …
+  /foo/{p1}:
+    …
+```
+
+| Client URL | `$request/path` | `$request/endpoint` |
+| --- | --- | --- |
+| https://example.com/api/foo/qux | /api/foo/qux | /api/foo/qux |
+| https://example.com/api/foo/quuux | /api/foo/quuux | /api/foo/quuux |
+| https://example.com/api/foo/bar | /api/foo/bar | /api/foo |
```

Comment:
Wo ist bei den beiden der Unterschied?

Reply:
Bei welchen beiden?

Reply:
Ah, verstehe :-)

Reply:
Besser
```
| https://example.com/api/foo/bar/qux | /api/foo/bar/qux | /api/foo |
```
?

Reply:
Ja! Vielleicht noch zur Verdeutlichung eine Spalte mit dem gematchten Endpunkt in die Tabelle? Alternativ im Swagger die gematchten URIs als Kommentare.

Reply:
| Client URL | matches | `$request/path` | `$request/endpoint` |
| --- | --- | --- | --- |
| https://example.com/api/foo/qux | /foo/qux | /api/foo/qux | /api/foo/qux |
| https://example.com/api/foo/quuux | /foo/{p1} | /api/foo/quuux | /api/foo/quuux |
| https://example.com/api/foo/bar/qux | /foo/** | /api/foo/bar/qux | /api/foo |
| https://example.com/api/bar | /** | /api/bar | /api |

Reply:
:+1: